docs/pages/database-access/faq.mdx

```diff
@@ -17,8 +23,19 @@ The following PostgreSQL protocol features aren't currently supported:
   Cancel requests issued by the PostgreSQL clients connected to Teleport proxy
   won't be passed to the database server.
 - Any of the [authentication methods](https://www.postgresql.org/docs/current/auth-methods.html)
-  except for client certificate authentication and IAM authentication for AWS
-  RDS and Aurora databases.
+  except for client certificate authentication and IAM authentication for cloud
+  databases.
+
+## Can database clients use public address different from web public address?
+
+Set `postgres_public_addr` and `mysql_public_addr` proxy configuration fields to
+public addresses over which respective database clients should connect. See
+[Proxy Configuration](./reference/configuration.mdx#proxy-configuration) for
+more details.
+
+This is useful when Teleport web proxy UI is running behind an L7 load balancer
+(e.g. ALB in AWS) in which case PostgreSQL/MySQL proxy need to be exposed on a
+plain TCP load balancer (e.g. NLB in AWS).
 
 ## Do you support X database client?
 
```

Comment:
Have been reviewing these and validating steps. Overall seems great!

QQ - AWS RDS now has a `serverless` option built on top of MySQL but with some slight differences. Anything we need to say about this here?

Reply:
@inertial-frame Good question - serverless RDS/Aurora is not supported because it currently does not support IAM authentication. I'll add a note here (and maybe on the respective guide pages as well).

Reply:
Added notes to FAQ and Postgres/MySQL AWS guides.